command-expansion-server/cdict/command_banananation.xml

```diff
@@ -70,9 +86,9 @@
                     <range_of_values>
                         <include min="0" max="100" />
                     </range_of_values>
-                    <description>Number of bananans to grow</description>
+                    <description>Number of bananas to grow</description>
                 </unsigned_arg>
-                <unsigned_arg name="duration" bit_length="8" units="none">
+                <unsigned_arg name="durationS" bit_length="8" units="none">
```

Comment:
```suggestion
                <unsigned_arg name="durations" bit_length="8" units="none">
```

Reply:
This was intended as duration-in-seconds, not a plural of duration. Not sure how better to annotate that in the naming. I know durationMs is super obvious, but less human friendly than seconds.

Reply:
Ahh gotcha, `durationSecs` or something could be clearer at first glance, but I also think it's fine as is then.